sum.js

```diff
@@ -0,0 +1,7 @@
+const range = (start = 0, end) => {
+    return [...Array(end - start).keys()].map(i => i + start);
```

Comment:
Oh, I forgot that `Array` have `keys` method, nice. (Also, you can omit `return` and braces)